app/components/gh-markdown-editor.js

```diff
@@ -311,7 +311,7 @@ export default Component.extend(ShortcutsMixin, {
         delete this._onEditorPaneScroll;
     },
 
-    _openImageFileDialog({captureSelection = true}) {
+    _openImageFileDialog(captureSelection = true) {
```

Comment:
FYI, this is a great walkthrough of named parameters in es6 http://exploringjs.com/es6/ch_parameter-handling.html#sec_named-parameters